src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsPatternVariables.java

```diff
@@ -0,0 +1,35 @@
+/*
+SuppressWarnings
+format = ^unchecked$*|^unused$*|.*
```

Comment:
I am not seeing anything that goes against this format. Can you just add 1 example?

Also, isn't `$*` basically invalid? Shouldn't this format be `^(unchecked|^unused|).*$`?

Seems like the last one will match everything, so you can't go against the format at all. I would still like to see one counter example to ensure this is processing correctly.

Reply:
Yea sorry this regex will match everything I updated it and added examples that goes against it